docs/compass/03-01-security.md

```diff
@@ -38,22 +32,22 @@ The table is used by Director and Tenant Mapping Handler. It contains the follow
 - `app_id` foreign key of type UUID
 - `runtime_id` foreign key of type UUID
 - `integration_system_id` foreign key of type UUID
-- `value` of type JSONB (with authentication details, such as `client_id/client_secret` in OAuth 2.0 authentication flow, `username/password` in Basic authentication flow; in case of certificates flow it is empty)
+- `value` of type JSONB (with authentication details, such as `client_id/client_secret` in OAuth 2.0 authentication flow, `username/password` in the basic authentication flow; in case of certificates flow it is empty)
 
 ### GraphQL security
 
-The Gateway passes request along with JWT token to Compass GraphQL services, such as Director or Connector. The GraphQL components have authentication middleware and GraphQL [directives](https://graphql.org/learn/queries/#directives) set up for all GraphQL operations (and some specific type fields, if necessary).
+Gateway passes a request along with a JWT to Compass GraphQL services, such as the Director or Connector. The GraphQL components have authentication middleware and GraphQL [directives](https://graphql.org/learn/queries/#directives) set up for all GraphQL operations (and some specific type fields, if necessary).
 
 #### HTTP middleware
 
-In GraphQL servers, such as Director or Connector, there is a HTTP authentication middleware set up, which validates and decodes JWT token. It puts user scopes and tenant in request context
+In GraphQL servers, such as the Director or Connector, there is an HTTP authentication middleware set up, which validates and decodes JWTs. It puts user scopes and tenant in request context
```

Comment:
The middleware for Connector doesn't validate JWT tokens. Client-Id-From-Token or Client-Id-From-Token header is used instead.